Pocket/pocket-ios
# Core Data Migration - Add mapping from PocketModel to PocketModel-3

## Summary
*This PR adds a mapping for users that will migrate from `PocketModel` (currently in production) to `PocketModel 3`

## References 
* NA

## Implementation Details
* Add a new mapping model

## Test Steps
* Install release `8.0.5.25060`
* Update to this branch and make sure the migration succeeds, you don't see any crashes at launch and the app behaves as expected
* Alternatively, install the app from the App Store on a device, then update to the TestFlight build `25406` and make sure everything looks good.

## PR Checklist:

## Screenshots
